samples/CommunityToolkit.Maui.Sample/MauiProgram.cs

```diff
@@ -68,9 +68,13 @@ public static MauiApp CreateMauiApp()
 								.UseMauiCommunityToolkitCamera()
 								.UseMauiCommunityToolkitMediaElement()
 
```

Comment:
MacCatalyst?

Reply:
I'm unable to validate that. 
In addition, the 2.0.1 release notes to CommunityToolkit.Maui.Maps only referenced WINDOWS, IOS, and ANDROID.

[CommunityToolkit.Maui.Maps 2
0.1 Release Notes](https://github.com/CommunityToolkit/Maui/releases/tag/2.0.1-maps)

Reply:
Please add MacCatalyst. It works.

Reply:
Thanks for the feedback. Pull Request amended.